www/source/partials/docs/_dev-pkgs-plan-builds.html.md.erb

```diff
@@ -49,6 +47,14 @@ The directory where your plan is located is known as the plan context.
 
 4. If the package builds successfully, it is placed into a `results` directory at the same level as your plan.
 
+### Managing Studio flavor to enter (Docker/Linux/Windows)
```

Comment:
I didn't quite understand what this meant. Maybe consider `Managing the Type of Studio to Use` or `Enter`?